data/src/main/java/pl/edu/agh/model/Project.java

```diff
@@ -1,5 +1,36 @@
 package pl.edu.agh.model;
 
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
 public class Project {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @Column(length = 64)
+    private String name;
+    @Transient //TODO add missing mapping
+    private List<User> usersInProject;
```

Comment:
Czy chcemy zmienić nazwę ```usersInProject``` z uwagi na to, że sprinty w projekcie to po prostu ```sprints```?

Reply:
UserInProject ma u nas inne znaczenie, wiec pewnie tak